.mailmap

```diff
@@ -0,0 +1,11 @@
+Antoine Petty <a.petty@ncl.ac.uk> <36314396+antoinepetty@users.noreply.github.com>
```

Comment:
```suggestion
Antoine Petty <github@antoinepetty.co.uk>
Antoine Petty <github@antoinepetty.co.uk> <36314396+antoinepetty@users.noreply.github.com>
```

Reply:
Is this to attribute people to the repo? I don't really understand

Reply:
Currently if you run `git log` in this repo you get lots of different names for what are actually the same people. This is most evident when running `git shortlog --summary --email`.

Adding this `.mailmap` file tells git which of those identities are in fact the same person as well as what true name to use for them.

Reply:
The `@users.noreply.github.com` addresses come from when you use GitHub's UI to make changes. You can fix that by setting which of your email addresses GitHub should use for those commits.

Reply:
Ah so my suggested changes are actually very unhelpful??

Reply:
Sorry, no, they were useful! I wanted to force-push to remove the original commit so that your other email address wasn't part of the history here. I'd copied that from the docs repo as I'd assumed it was your preferred one.